hudi-utilities/src/test/java/org/apache/hudi/utilities/deltastreamer/TestHoodieDeltaStreamerSchemaEvolutionQuick.java

```diff
@@ -59,25 +59,34 @@ public void teardown() throws Exception {
   }
 
   protected static Stream<Arguments> testArgs() {
+    boolean fullTest = false;
     Stream.Builder<Arguments> b = Stream.builder();
-    //only testing row-writer enabled for now
-    for (Boolean rowWriterEnable : new Boolean[] {true}) {
-      for (Boolean nullForDeletedCols : new Boolean[] {false, true}) {
-        for (Boolean useKafkaSource : new Boolean[] {false, true}) {
-          for (Boolean addFilegroups : new Boolean[] {false, true}) {
-            for (Boolean multiLogFiles : new Boolean[] {false, true}) {
-              for (Boolean shouldCluster : new Boolean[] {false, true}) {
-                for (String tableType : new String[] {"COPY_ON_WRITE", "MERGE_ON_READ"}) {
-                  if (!multiLogFiles || tableType.equals("MERGE_ON_READ")) {
-                    b.add(Arguments.of(tableType, shouldCluster, false, rowWriterEnable, addFilegroups, multiLogFiles, useKafkaSource, nullForDeletedCols));
+    if (fullTest) {
+      //only testing row-writer enabled for now
+      for (Boolean rowWriterEnable : new Boolean[] {true}) {
+        for (Boolean nullForDeletedCols : new Boolean[] {false, true}) {
+          for (Boolean useKafkaSource : new Boolean[] {false, true}) {
+            for (Boolean addFilegroups : new Boolean[] {false, true}) {
+              for (Boolean multiLogFiles : new Boolean[] {false, true}) {
+                for (Boolean shouldCluster : new Boolean[] {false, true}) {
+                  for (String tableType : new String[] {"COPY_ON_WRITE", "MERGE_ON_READ"}) {
+                    if (!multiLogFiles || tableType.equals("MERGE_ON_READ")) {
+                      b.add(Arguments.of(tableType, shouldCluster, false, rowWriterEnable, addFilegroups, multiLogFiles, useKafkaSource, nullForDeletedCols));
+                    }
                   }
                 }
+                b.add(Arguments.of("MERGE_ON_READ", false, true, rowWriterEnable, addFilegroups, multiLogFiles, useKafkaSource, nullForDeletedCols));
               }
-              b.add(Arguments.of("MERGE_ON_READ", false, true, rowWriterEnable, addFilegroups, multiLogFiles, useKafkaSource, nullForDeletedCols));
             }
           }
         }
       }
+    } else {
```

Comment:
```
                          String tableType = COW, MOR
                          Boolean shouldCluster = true
                          Boolean shouldCompact = true
                          Boolean rowWriterEnable = true
                          Boolean addFilegroups = true
                          Boolean multiLogFiles = true
                          Boolean useKafkaSource= false, true
                          Boolean allowNullForDeletedCols=false,true
                          ```
                          
                          I wonder if we just do sth like this. with new file groups, multiple log files, alongside cluster and compaction, should be the more complex (superset) scenario. no?